### ISSUE TITLE
Parse errors in imported module 'marked': Unexpected token = (2475:27)  import/namespace

### ISSUE THREAD

**Issue by webketje** [import-js/eslint-plugin-import#2791]
Context: Node 18, native ES module source, but output to both `esm.js` and `.cjs`.
When importing marked@^4.3.0:

```js
import { marked } from 'marked'
```
it looks like eslint-plugin-import parser fails on syntax for **static class properties** (not methods): (see https://github.com/markedjs/marked/blob/v4.3.0/lib/marked.esm.js#L2475) and erronously throws.
The rule thrown is https://github.com/import-js/eslint-plugin-import/blob/v2.27.5/docs/rules/namespace.md

Workaround: set `import/namespace` to `off` in `.eslintrc.yml`

**Comment by ljharb** [comment]
It shouldn't; what's your eslint config? For the default parser, ecmaVersion needs to be set high enough for static class fields to be understood.

**Comment by webketje** [comment]
Ah ok, tried different Node vers but no impact. As you say, ecmaVersion was the issue (it was set to 2020, and needs to be set at least to 2022). Though it is really hard to trace when the static class fields proposal became part of the official ECMAscript spec :(

**Comment by ljharb** [comment]
https://github.com/tc39/proposals/blob/main/finished-proposals.md
